WaterDock-2.0_Basics.md

```diff
@@ -0,0 +1,76 @@
+**This file is a more detailed version of README.md.**
+N.B. The command line commands in this text file work for Mac but may not work for Windows and Linux. You might need to look up the equivalent commands for these operating systems.
+
+HOW TO SET UP WATERDOCK-2.0 (DETAILED)
+
+1.   Download the zip file or clone using GitHub.
+
+
+2.   Have Python 3.7 (or the latest version of Python 3 installed onto your computer).
+
+
+3.   Ensure that you have the MDAnalysis, numpy and scipy packages installed using either pip or conda.
+      - Look up the correct commands for installing each package using Google\

```

Comment:
A thought here, maybe just for a future iteration of this file. Rather than telling people to Google things, I think we can go a step up and provide them with a framework to install things.

Here we can tell people to just use conda. We can add an `environment.yml` file to the repository that allows them to build a conda environment easiliy. See here for an example of such a file: https://github.com/RMeli/spyrmsd/blob/develop/devtools/conda-envs/spyrmsd.yaml

A minimal environment.yaml file would just contain:
```yaml
name: waterdock2
channels:
  - conda-forge
dependencies:
  - MDAnalysis
```
Note; numpy and scipy are core dependencies of MDAnalysis so you don't really need to have them as extra dependencies.

Then here you'd just have to tell users to do:

```bash
conda env create python=3.8 environment.yaml
conda activate waterdock2
```

Reply:
That would be useful - I forgot to add to the instructions that `pip install -r requirements.txt` can be used for pip dependencies so if this yaml file was created, we would be accommodating both pip and conda and be able to provide both possible instructions. I might be wrong on this though.

Reply:
Ideally it'd be even easier if we just had a setup.py file in place. That way you just need to do `pip install .` and it will pick up the extra dependencies. This is probably something that should be done in the future though.